app/assets/stylesheets/jobs.css

```diff
@@ -0,0 +1,24 @@
+.jobs {
+  float: right;
+  width: 80%;
+}
+
+table {
+  width: 100%;
+}
+
+.index-job {
+  width: 100%;
+  border: solid;
+  border-color: black;
+  border-width: 1px;
+}
+
+.index-job:hover {
+  background-color: #BDC3C7;
+}
+
+.index-job-detail {
+  padding-left: 10px;
+  padding-right: 10px;
```

Comment:
Honestly, I like to see them broken out so it is clear to the next person what is being manipulated. I would have to look it up to see in what direction that single line was pushing things. I'm open to changing it though